### PR TITLE
testing/acsccid: new aport

### PR DESCRIPTION
**Pull request by misery**

### CODE REVIEW

File: testing/acsccid/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+pkgname=acsccid
+pkgver=1.1.4
+pkgrel=0
+pkgdesc="PCSC driver for ACS CCID smart card reader"
+url="https://github.com/acshk/acsccid"
+arch="all"
+license="LGPL"
+depends="pcsc-lite"
+makedepends="pcsc-lite-dev libusb-dev perl flex"
+options="!check"
```
Comment (jirutka) [inline testing/acsccid/APKBUILD]: Please write a reason (as a comment on this line) why check is disabled (upstream does not provide tests?).

File: testing/acsccid/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="${pkgdir}" install
+	install -Dm644 src/92_pcscd_acsccid.rules "$pkgdir/etc/udev/rules.d/92_pcscd_acsccid.rules"
```
Comment (jirutka) [inline testing/acsccid/APKBUILD]: Should be udev rules really installed to `/etc`? /cc @kaniini @Shizmob

File: testing/acsccid/APKBUILD (hunk 1 above)
Comment (misery) [inline testing/acsccid/APKBUILD]: done ... there is an empty "make check" only. No tests provided

File: testing/acsccid/APKBUILD (hunk 2 above)
Comment (Shizmob) [inline testing/acsccid/APKBUILD]: I'd like a `-udev` subpackage with install_if=udev here personally, if the package doesn't inherently depend on udev.

Comment (misery) [inline testing/acsccid/APKBUILD]: done

**Comment by misery** [comment]
ping :-)

**Comment by misery** [comment]
Any more changes?

File: testing/acsccid/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="${pkgdir}" install
+}
+
+udev() {
+	pkgdesc="$pkgdesc (udev rules)"
+	install_if="udev"
```
Comment (jirutka) [inline testing/acsccid/APKBUILD]: There must be `install_if="$pkgname=$pkgver-r$pkgrel udev"`.

Comment (misery) [inline testing/acsccid/APKBUILD]: thanks! Done...

**Comment by misery** [comment]
More changes? :-)

**Pull request closed without merging**